app/src/main/res/layout/activity_edit_profile.xml

```diff
@@ -14,77 +14,82 @@
         android:layout_height="match_parent"
         app:layout_behavior="@string/appbar_scrolling_view_behavior">
 
-        <LinearLayout
+        <androidx.constraintlayout.widget.ConstraintLayout
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:focusableInTouchMode="true"
-            android:orientation="vertical">
+            android:focusableInTouchMode="true">
 
-            <RelativeLayout
+            <ImageView
+                android:id="@+id/headerPreview"
                 android:layout_width="match_parent"
-                android:layout_height="wrap_content">
-
-                <ImageView
-                    android:id="@+id/headerPreview"
-                    android:layout_width="match_parent"
-                    android:layout_height="200dp"
-                    android:contentDescription="@null" />
-
-                <ImageButton
-                    android:id="@+id/headerButton"
-                    android:layout_width="match_parent"
-                    android:layout_height="200dp"
-                    android:background="#66000000"
-                    android:contentDescription="@string/label_header"
-                    app:srcCompat="@drawable/ic_add_a_photo_32dp" />
-
-                <ProgressBar
-                    android:id="@+id/headerProgressBar"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:layout_centerInParent="true"
-                    android:indeterminate="true"
-                    android:visibility="gone" />
-
-            </RelativeLayout>
+                android:layout_height="200dp"
+                android:contentDescription="@null"
+                app:layout_constraintTop_toTopOf="parent" />
 
+            <ImageButton
+                android:id="@+id/headerButton"
+                android:layout_width="match_parent"
+                android:layout_height="200dp"
+                android:background="#66000000"
+                android:contentDescription="@string/label_header"
+                app:layout_constraintTop_toTopOf="parent"
+                app:srcCompat="@drawable/ic_add_a_photo_32dp" />
+
+            <ProgressBar
+                android:id="@+id/headerProgressBar"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:indeterminate="true"
+                android:visibility="gone"
+                app:layout_constraintBottom_toBottomOf="@id/headerPreview"
+                app:layout_constraintEnd_toEndOf="parent"
```

Comment:
If you want to center it in the header, maybe say it explicitly and not to the parent?